### DISCUSSION TITLE
Add chapter support

### DISCUSSION THREAD

**Discussion by AshtyBoi** [Rehike/Rehike#581] [Ideas]
Because YouTube didn't have chapters back when Hitchhiker was a thing, the "View Chapter" button on videos with chapters does nothing.
![image](https://user-images.githubusercontent.com/107463768/188567901-64da1b7f-7468-46fe-9572-e83b3ac21c8d.png)
How the Chapters menu looks on Polymer

Now, this isn't really an issue, but adding support for this feature would be convenient for people who use the Chapters feature.

**Comment by ghost** [comment]
This is in the plans, although it will not be implemented for a while as other things need to be worked on first.

**Comment by alpharetzy** [comment]
This is a mockup i made in PowerPoint showing the chapters feature in Hitchhiker layout:

![Screenshot (980)](https://user-images.githubusercontent.com/112749084/206185686-dbcf16dd-6dac-4cf8-bfcd-5ec9d9f72ff2.png)

**Comment by alpharetzy** [comment]
Here's the earlier variant i made with the load more button to show more chapters and the sync to video time button visible:

![Screenshot (979)](https://user-images.githubusercontent.com/112749084/206710671-08b2d28d-f9a8-44a0-a58c-47dddba39612.png)

At this point, do you prefer the top or bottom one?

**Comment by THErlego** [comment]
so far liking top more
its very unlikely  for a video to have so many chapters that they cant be all loaded at once (standard playlist limit is 100 elements at one time)
just wondering whats the repeat button do?

**Comment by alpharetzy** [comment]
The repeat button will set to loop the chapter and will NOT jump to the next chapter, instead it will go back to the start of the said chapter.

**Comment by alpharetzy** [comment]
It’s now a year later, why isn’t this a thing yet??

**Comment by ghost** [comment]
> It’s now a year later, why isn’t this a thing yet??

We are lazy.
